### ISSUE TITLE
Installation error: Super expression must be either null or a function not undefined

### ISSUE THREAD

**Issue by homanp** [xotahal/react-native-material-ui#59]
Hi guys,

Getting this issue when trying to import Toolbar: 

`super expression must be either null or a function not undefined`

Doing it this way:

`import { Toolbar } from 'react-native-material-ui';` in my component. 

Any ideas?

**Comment by xotahal** [comment]
What version of react and react-native do you have?

**Comment by homanp** [comment]
I run React 15.1  and RN 0.27.2

**Comment by xotahal** [comment]
There could be problem with `PureComponent` which was added in [React 15.3.0](https://github.com/facebook/react/releases/tag/v15.3.0). Could you try it with that version of react?

**Comment by homanp** [comment]
Ok! Trying 15.3, get back to you with results.

**Comment by xotahal** [comment]
And also, there were a couple of changes in React Native (current version is 0.39), so you could have problem with `flex` layout.

**Comment by homanp** [comment]
Do you support 0.27?

**Comment by homanp** [comment]
Yep, 15.3 did solve the error but that requires me to update RN to > 0.30. Do you by any chance support earlier versions of RN?

**Comment by xotahal** [comment]
I think it's not possible to support it. There were some breaking changes with layout in React Native. Is there any problem to upgrade to RN 0.39?
